.gitignore

```diff
@@ -4,5 +4,4 @@
 /build
 *.iml
 *.ipr
-*.iws
-/podman/production.conf
\ No newline at end of file
+*.iws
```

Comment:
You removed the `/podman/production.conf` from gitignore, but it is an important file that we don't want committed (the production config file one can use for the cloud deployment). Please add back.